src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx

```diff
@@ -148,8 +148,8 @@ export default class IntegrationRepos extends AsyncComponent {
     let repositories = this.state.integrationRepos.repos;
     let items = (repositories || []).map(repo => {
       return {
-        searchKey: `${repo.name}`,
-        value: `${repo.full_name}`,
+        searchKey: repo.name,
+        value: repo.identifier,
```

Comment:
We need to change the GitHub provider for this too right? I think you mentioned this in person today @MeredithAnya

Reply:
Yup! It'll break it for a moment. We can get that in soon after.